src/diagnosticMessages.json

```diff
@@ -184,6 +184,7 @@
   "Type '{0}' has no call signatures.": 2757,
 
   "File '{0}' not found.": 6054,
+  "no such global element: '{0}'":6055,
```

Comment:
Diagnostic 6055 is [different in TypeScript](https://github.com/microsoft/TypeScript/blob/073ac920adee8e517350a455908aa420b29f9422/src/compiler/diagnosticMessages.json#L4260-L4263), and we try to prevent collisions there. What seems to fit, though, is to renumber this diagnostic to 110 and move it up to the 1XX range of custom AS compiler errors.

Error ranges are about:
* 100-199: Custom AS compiler errors
* 200-899: Custom AS compilation diagnostics
* 900-999: Pedantic diagnostics
* 1000+: Reused TypeScript errors